app/main/views/frameworks.py

```diff
@@ -34,15 +34,68 @@ def opportunities_dashboard(framework_slug):
     # Split into two tables by status
     drafts, completed = [], []
     two_weeks_ago = (datetime.now() - timedelta(days=14))
+
     for opportunity in opportunities:
+        brief = opportunity.get('brief')
+        applicationsClosedAt = brief.get("applicationsClosedAt")
+
         if opportunity['status'] == 'draft':
+            opportunity_row = [
+                {"text": brief.get('title')},
+                {"text": dateformat(applicationsClosedAt), "attributes": {"data-closed": applicationsClosedAt}}
+            ]
+            opportunity_row.append({"text": "Draft"})
```

Comment:
I don't think the append is needed here?

```suggestion
            opportunity_row = [
                {"text": brief.get('title')},
                {"text": dateformat(applicationsClosedAt), "attributes": {"data-closed": applicationsClosedAt}},
                {"text": "Draft"},
            ]
```

Reply:
Ah, good spot - from an earlier draft when I had `opportunity_row` defined in the outer `for` loop.

fixed.